### PR TITLE
delays air initialization to after the entire spaceruin has loaded

### PR DESCRIPTION
**Pull request by jupyterkat**
## About The Pull Request

see title

## Why It's Good For The Game

because each template gets atmos inited in a separate order, of course some tiles are gonna get vented to space where they shouldn't

## Changelog
:cl:
fix: fixed ruin turfs get vented sometimes
/:cl:

### CODE REVIEW

File: code/controllers/subsystem/air.dm [hunk 1]
```diff
@@ -429,8 +429,20 @@ SUBSYSTEM_DEF(air)
 
 /datum/controller/subsystem/air/proc/pause_z(z_level)
 	LAZYADD(paused_z_levels, z_level)
+	var/list/turfs_to_disable = block(locate(1, 1, z_level), locate(world.maxx, world.maxy, z_level))
+	for(var/thin in turfs_to_disable)
+		var/turf/T = thin
```
Comment (ike709) [inline code/controllers/subsystem/air.dm]:
```suggestion
	for(var/turf/T as anything in turfs_to_disable)
```

File: code/controllers/subsystem/air.dm [hunk 2]
```diff
@@ -429,8 +429,20 @@ SUBSYSTEM_DEF(air)
...
+		T.atmos_adjacent_turfs?.Cut()
+		CHECK_TICK
 
 /datum/controller/subsystem/air/proc/unpause_z(z_level)
+	var/list/turfs_to_reinit = block(locate(1, 1, z_level), locate(world.maxx, world.maxy, z_level))
+	for(var/thin in turfs_to_reinit)
+		var/turf/T = thin
```
Comment (ike709) [inline code/controllers/subsystem/air.dm]:
```suggestion
	for(var/turf/T as anything in turfs_to_reinit)
```

**Comment by PowerfulBacon** [comment]
I have no idea how atmos works

**Comment by jupyterkat** [comment]
shame

**Review by PowerfulBacon — DISMISSED** [review]
Seems good I think?
As long as its tested, last time I tried touching SSatomspherics with basic pause_z proc it caused the entire station to get maxcapped

**Review by Crossedfall — APPROVED** [review]
No issues reported during the TM.

**Pull request merged**